crmsh/bootstrap.py

```diff
@@ -1375,136 +1536,14 @@ def sysconfig_comment_out(scfile, key):
     return outp, ncomments
 
 
-def init_sbd_diskless():
-    """
-    Initialize SBD in diskless mode.
-    """
-    status_long("Initializing diskless SBD...")
-    if os.path.isfile(SYSCONFIG_SBD):
-        log("Overwriting {} with diskless configuration".format(SYSCONFIG_SBD))
-        scfg, nmatches = sysconfig_comment_out(open(SYSCONFIG_SBD), "SBD_DEVICE")
-        if nmatches > 0:
-            utils.str2file(scfg, SYSCONFIG_SBD)
-    else:
-        log("Creating {} with diskless configuration".format(SYSCONFIG_SBD))
-    utils.sysconfig_set(SYSCONFIG_SBD,
-                        SBD_PACEMAKER="yes",
-                        SBD_STARTMODE="always",
-                        SBD_DELAY_START="no",
-                        SBD_WATCHDOG_DEV=detect_watchdog_device())
-    csync2_update(SYSCONFIG_SBD)
-    status_done()
-
-
 def init_sbd():
```

Comment:
I think we can remove this method, right?

Reply:
Currently, we can't do that. `init_xxx` was a format for `stage`, used by `globals()["init_" + stage]()`

Reply:
Ok, I see. Thanks for the clarification